hibernate-core/src/test/java/org/hibernate/orm/test/query/hql/FunctionTests.java

```diff
@@ -77,7 +78,7 @@ public class FunctionTests {
 
 	public static final double ERROR = 0.00001d;
 
-	@BeforeAll @SuppressWarnings("deprecation")
+	@BeforeEach @SuppressWarnings("deprecation")
```

Comment:
There is false positive tests with Informix. Look at https://hibernate.zulipchat.com/#narrow/stream/132096-hibernate-user/topic/Tests.20with.20data.20prepared.20in.20.40BeforeAll

Reply:
Tables and data are truncated once a test fails, that's true - but the point of tests is that none should fail. Instead of changing the data preparation, you should fix or skip all tests that are failing in that class.

Reply:
Okay, I'll change that. In the end, it's true. However, partial fixes won't be considered on an ongoing basis.